aptos-move/framework/supra-framework/sources/genesis.move

```diff
@@ -188,6 +189,7 @@ module supra_framework::genesis {
         reconfiguration::initialize(&supra_framework_account);
         block::initialize(&supra_framework_account, epoch_interval_microsecs);
         state_storage::initialize(&supra_framework_account);
+        automation_registry::initialize(&supra_framework_account);
```

Comment:
I don't think we should call this here. Otherwise replay of the chain would not agree on its state. Since we have launched mainnet, this should not be touched now. `automation_registry::initialize` can be called via governance script (upgrade framework followed by call to initialize)

Reply:
The testing will not be easy if we do not have initialize called here. as during testing we start the chain from genesis.
I remember you once mentioned that you want to avoid conditional initialization, but at this moment I do not see easy way to have the module initialized properly.